File: cpp/40b_1729A_TwoElevators.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, c, cnt1, cnt2;
    std::vector<int> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>a>>b>>c;
        cnt1 = abs(a - 1);
        cnt2 = abs(c - b) + abs(c - 1);
        if (cnt1 < cnt2){
            ans.push_back(1);
        }
        else if(cnt1 > cnt2){
            ans.push_back(2);
        }
        else{
            ans.push_back(3);
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/40b_1729A_TwoElevators.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        a, b, c = input().split()
        a, b, c = int(a), int(b), int(c)
        cnt1 = abs(a - 1)
        cnt2 = abs(c - b) + abs(c - 1)
        if cnt1 > cnt2:
            ans.append(2)
        elif cnt1 < cnt2:
            ans.append(1)
        else:
            ans.append(3)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()